eos-tech-support/eos-install-firefox

```diff
@@ -0,0 +1,341 @@
+#!/bin/bash -e
+
+# Downloads Firefox and unzips to the user's home directory,
+# with a desktop launcher
+
+exit_with_error() {
+   echo $1
+   exit 1
+}
+
+USERID=$(id -u)
+if [ "$USERID" == "0" ]; then
+    exit_with_error "Program cannot be run as superuser"
+fi
+
+DEB_ARCH=`dpkg --print-architecture`
+case ${DEB_ARCH} in
+    amd64) ;; # Only 64-bit Intel architectures are supported
+    *) exit_with_error "Unsupported architecture ${DEB_ARCH}"
+       ;;
+esac
+
+TEMP_DIR=`mktemp -d`
+DEST_DIR=~/.firefox
```

Comment:
See the other comment on the Chrome PR. Basically use another directory inside the XDG ones, e.g. `~/.local/share/eos-third-party-apps`